### DISCUSSION TITLE
[FEATURE-REQUEST] ¿Se puede incluir en el admin editar subordinates?

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#2] [3. Ideas & Feature Requests] [answered]
Hola a todos!

En implementaciones anteriores, tenemos el uso de **hierarchy.subordinates** que actualmente funciona en las questions de tipo selector de usuarios, pero los seteamos a través de la API. 

Ayudaría mucho si esta configuración se puede hacer en el administrador directamente ya que un cliente necesita gestionar sus propios usuarios utilizando esta herramienta visual. 

Saludos!

**Comment by guillevalin** [comment, marked as answer]
@SeIturra por favor no incluir información de compañías ya que esto es un medio de comunicación público.

Respecto a la funcionalidad de subordinados, existe una inconsistencia entre el selector del admin, versus la configuración del formulario, la cual se normalizará. Por favor levantar un incidente.

**Reply by seiturra** [reply]
Ok, entonces configuro todo con subordinates y comunicaré al cliente que esto estará disponible "pronto".
Comprendo que debería estar antes de un mes, pero la entrega es la próxima semana así que por ahora la carga de usuarios la realizaremos a mano por BD.

**Reply by seiturra** [reply]
#2643
